wire-tests/src/commonTest/proto/kotlin/swift_edge_cases.proto

```diff
@@ -37,6 +37,11 @@ message Screen {
   oneof screen {
     string oneof_string = 1;
     int32 oneof_int32 = 2;
+    SubMessage oneof_sub_message = 3;
```

Comment:
Compiling generated Swift code without the change in the SwiftGenerator above fails. 

Also verified in our internal repo this fixes this edge case.

Reply:
Also I should note there are other protos that do not have the same name as the enclosing message name that from this change do not use the `_OneOf` suffix. This suffix is only used in this edge case.